genomic_data_service/regulome_atlas.py

```diff
@@ -416,6 +430,21 @@ def _score(characterization):
         query = [int(k in characterization) for k in binary_keys]
         numeric_keys = ['IC_max', 'IC_matched_max']
         query += [characterization[k] for k in numeric_keys]
+        tissue_specific_query = {}
+        for organ in ORGANS:
+            tissue_specific_query[organ] = [0]*7
```

Comment:
it's a little weird to store this as a "binary feature vector" in python, normally you would use a dict but I suppose you would convert it anyway.